documentation/faq/10-version-numbers.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: SvelteKit is not 1.0 yet. Should I use it? What about Sapper?
+title: SvelteKit is not 1.0 yet. Should I use it?
 ---
 
-SvelteKit is currently in beta while we finalize the APIs. No new features will be added to Sapper and all development will be focused on SvelteKit.
+SvelteKit is currently in release candidate phase. We expect minimal breaking changes and for development to be focused on bug fixes.
```

Comment:
Should we go "minimal to no"? At least that's my hope at this point 😅

Reply:
updated :smile: